### DISCUSSION TITLE
Currency Conversion Symbols

### DISCUSSION THREAD

**Discussion by MitchellGriggsTAFE** [MitchellGriggsTAFE/TafeCalc24#2] [Ideas]
I noticed the Currency Conversion Calculator template interchanges between currency abbreviations (i.e. USD, GPD, EUR) and full text (i.e. US Dollars, Great British Pound, Euros). To ensure consistency and user friendliness, I was thinking it might be a good idea to simply give both forms in the From/To input menu, and then only use abbreviations in the output results?

**Comment by TravisGriggsProjects** [comment]
I completely agree. It's important we ensure consistency and brevity so that users have the best experience possible.
